### DISCUSSION TITLE
Changelogs

### DISCUSSION THREAD

**Discussion by Fr-Dae** [audacious-media-player/audacious#1518] [General]
Hello, 
I just woke up from version 22.04 of ubuntu (4.1) and I'm delighted to see this fantastic program still live and active (thanks to the ppa https://launchpad.net/~ubuntuhandbook1 ) 
on the other hand, I can't find where to consult the update changes between version 4.4 and version 4.1, I'd like to catch up.

Also, do you think it would be possible to add a ‘changelog’ button in ‘about’ to follow this one directly?  (like factorio does in-game)

**Comment by radioactiveman** [comment]
Release notes can be found on our [main page](https://audacious-media-player.org/) (linked already in the about dialog) and also on [GitHub](https://github.com/audacious-media-player/audacious/releases).

**Reply by Fr-Dae** [reply]
nothing before 4.4 on github , that my wuestion

**Comment by wildstar84** [comment]
If you're tecnhically-minded, you can also look at the commits at https://github.com/audacious-media-player/audacious[-plugins]/commits/master/ for the full gory details. ;)
